spec/src/modules/tracker.js

```diff
@@ -3667,6 +3667,344 @@ describe('ConstructorIO - Tracker', () => {
     });
   });
 
+
+  describe('trackGenericResultClick', () => {
+    // Note: `variation_id` parameter not being passed as none are defined
```

Comment:
What does `parameter not being passed as none are defined` mean? 🤔

Reply:
`variation_id` is a valid parameter that can be passed to this method, however none are currently defined in the catalog we use for testing, so we aren't ever passing it in testing

This comment definitely could be more clear - I'll update in this PR. Good callout @jjl014 !